app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -483,18 +483,17 @@ public boolean onLongClick(View view) {
     private void handleItemEvents(final WebSearchHolder websearchholder, final int position)  {
         websearchholder.backgroundLayout.setBackgroundColor(ContextCompat.getColor(currContext, isSelected(position) ? R.color.translucent_blue : android.R.color.transparent));
 
-        webquery = MainActivity.webSearch;
-
         final ChatMessage model = getData().get(position);
+        int checkIndex = model.getContent().lastIndexOf("#");
         websearchholder.messageStar.setVisibility( (model.isImportant()) ? View.VISIBLE : View.GONE);
-        websearchholder.text.setText(model.getContent());
+        websearchholder.text.setText(model.getContent().substring(0,checkIndex));
```

Comment:
Just confirmed with orbiter, the # will be interpreted as whitespace. So, most probably there won't be '#' in result. So, it may work. But, I am not that happy with overall implementation of websearch result. I am creating a new issue for improving it. This can be merged for now. Just resolve the conflicts.